dev-packages/e2e-tests/test-applications/astro-5/tests/tracing.dynamic.test.ts

```diff
@@ -62,7 +62,6 @@ test.describe('tracing in dynamically rendered (ssr) routes', () => {
     });
 
     expect(serverPageRequestTxn).toMatchObject({
-      breadcrumbs: expect.any(Array),
```

Comment:
this used to be the breadcrumbs of other errors from other tests.. which seems wrong to me, actually? So I wonder if this actually fixes something with astro, which is possible as now this is unrelated to when this is initialized, I guess..

Reply:
I guess this could fix some kind of isolation problem since request isolation in Astro was limited to the middleware lifecycle previously. Maybe something didn't work as expected in there or was a timing issue. Gonna say it's finde until proven otherwise :D